tests/snapshots/framework-detection.test.js.md

```diff
@@ -66,77 +64,100 @@ Generated by [AVA](https://avajs.dev).
 
     `◈ Netlify Dev ◈␊
-    ◈ "framework" option must be set to "#custom" when specifying both "command" and "targetPort" options`
+    ◈ Starting Netlify Dev with custom config␊
+    hello␊
+    ◈ "echo hello" exited with code *. Shutting down Netlify Dev server`
 
-## should error when a specific framework is configured but not detected
+## should force a specific framework when configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Specified "framework" detector "create-react-app" did not pass requirements for your project`
+    ◈ Starting Netlify Dev with Create React App␊
+    ◈ Failed launching framework server. Please verify 'react-scripts' exists`
 
 ## should throw when forcing a non supported framework
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Unsupported value provided for "framework" option in config. Please use "#custom" if you're using a framework not intrinsically supported by Netlify Dev. E.g. with "command" and "targetPort" options. Or use one of following values: "angular", "brunch", "create-react-app", "docusaurus", "eleventy", "ember", "expo", "gatsby", "gridsome", "hexo", "hugo", "jekyll", "middleman", "next", "nuxt", "parcel", "phenomic", "quasar-v0.17", "quasar", "react-static", "redwoodjs", "sapper", "stencil", "svelte-kit", "svelte", "vite", "vue", "vuepress"`
+    ◈ Invalid framework "to-infinity-and-beyond-js". It should be one of: docusaurus, docusaurus-v2, eleventy, gatsby, gridsome, hexo, hugo, jekyll, middleman, next-nx, next, blitz, nuxt, phenomic, react-static, redwoodjs, stencil, vuepress, assemble, docpad, harp, metalsmith, roots, wintersmith, cecil, zola, angular, create-react-app, ember, expo, quasar, quasar-v0.17, sapper, svelte, svelte-kit, vue, brunch, parcel, grunt, gulp, vite, wmr`
 
 ## should detect a known framework
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Starting Netlify Dev with create-react-app␊
+    ◈ Starting Netlify Dev with Create React App␊
     ␊
     > start␊
     > react-scripts start␊
     ␊
-    ◈ Command failed with exit code *: npm start. Shutting down Netlify Dev server`
+    ◈ Command failed with exit code *: npm run start. Shutting down Netlify Dev server`
 
 ## should throw if framework=#custom but command is missing
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ "command" and "targetPort" properties are required when "framework" is set to "#custom"`
+    ◈ 'command' and 'targetPort' properties are required when 'framework' is set to '#custom'`
 
 ## should throw if framework=#custom but targetPort is missing
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ "command" and "targetPort" properties are required when "framework" is set to "#custom"`
+    ◈ 'command' and 'targetPort' properties are required when 'framework' is set to '#custom'`
 
 ## should start custom command if framework=#custom, command and targetPort are configured
 
 > Snapshot 1
 
     `◈ Netlify Dev ◈␊
-    ◈ Overriding command with setting derived from netlify.toml [dev] block: echo hello␊
```

Comment:
This log message is wrong. The tests doesn't set the command via `[dev]` block, but via a flag:
https://github.com/netlify/cli/blob/08d87ff6b4785612ddf97e54098ced40d58bffb5/tests/framework-detection.test.js#L180